uk.co.compucorp.civicrm.hrcontactactionsmenu/hrcontactactionsmenu.php

```diff
@@ -162,3 +181,20 @@ function hrcontactactionsmenu_civicrm_alterSettingsFolders(&$metaDataFolders = N
   _hrcontactactionsmenu_civix_civicrm_alterSettingsFolders($metaDataFolders);
 }
 
+/**
+ * Checks whether the User account of the contact is
+ * disabled. Returns null if contact has no CMS user
+ * account
+ *
+ * @param array $contactInfo
+ *
+ * @return bool|null
+ */
+function _hrcontactactionsmenu_get_is_user_disabled($contactInfo) {
+  if(empty($contactInfo['id'])) {
+    return null;
```

Comment:
Well I have the impression that true means user account is disabled, false means its not and null means the condition check is not applicable to the user because contact does not have a user account. 
Adding false here won't affect the results though.